highton/models/attachment.py

```diff
@@ -22,3 +24,6 @@ def __init__(self, **kwargs):
         self.size = fields.IntegerField(name=HightonConstants.SIZE)
 
         super().__init__(**kwargs)
+
+    def get(self):
+        return self._get_request(endpoint=f'files/{self.id}', endpoint_suffix='').content
```

Comment:
Should also hand back the encoding. Maybe tuple it or just return the entire response? What do you think, @bykof ?